pulsar-broker/src/test/java/org/apache/pulsar/io/PulsarFunctionE2ETest.java

```diff
@@ -407,36 +407,6 @@ public void testAuthorization(boolean validRoleName) throws Exception {
         }
     }
 
-    /**
-     * Test to verify: function-server loads jar using file-url and derives type-args classes if not provided
-     * @throws Exception
-     */
-    @Test(timeOut = 20000)
-    public void testFileUrlFunctionWithoutPassingTypeArgs() throws Exception {
```

Comment:
why are we getting rid of this test?

Reply:
Because FunctionConfig has no notion of typeargs.

Reply:
but this tests if server side derives the type-args from the function:

Reply:
The thing is that as per the new api, we use SourceConfig for both input and output(that is for register/update as well as get) and SourceConfig does not have the concept of any types. Thus any E2E functionality cannot depend on any extraction of types at the backend(since there is really no way of getting that information). The only e2e testing should be whether the function is behaving in the way its expected to behave. Thus this particular unittest doesn't fit the bill.